code_examples/sdk_examples/src/core_features/did/08_full_did_batch.ts

```diff
@@ -1,17 +1,6 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
-
-function getRandomCType(): Kilt.ICType {
-  // Random factor ensures that each created CType is unique and does not already exist on chain.
-  const randomFactor = Kilt.Utils.UUID.generate()
-  return Kilt.CType.fromProperties(`CType ${randomFactor}`, {
-    name: {
-      type: 'string'
-    },
-    age: {
-      type: 'integer'
-    }
-  })
-}
+// Just a helper to get an extrinsic
+import getExtrinsic from '../utils/getExtrinsic'
 
 export async function batchCTypeCreationExtrinsics(
```

Comment:
Also this name is misleading. What does `batch` mean? What about `signAndSubmitCTypeCreationBatch`?

Reply:
good catch. We actually don't want to mention ctypes anymore. I renamed it to `signAndSubmitDidExtrinsicBatch`. So it also matches the other function. [ee24291](https://github.com/KILTprotocol/docs/pull/183/commits/ee24291b1852d2887938265dfcfb9efb876b0e48)